lib/index.js

```diff
@@ -58,6 +58,7 @@ var subtag = require("subtag");
  *   forwardGeocode: (config) => { return { features: [] } },
```

Comment:
My first time reviewing anything in this repo, so naive question. Any reason not to use TS in this package?

Reply:
Just legacy reasons, we forked it over from mapbox-gl-geocoder and simply haven't had enough cycles to migrate to using ts